updates/next.json

```diff
@@ -1,14 +1,14 @@
 {
   "stream": "next",
   "metadata": {
-    "last-modified": "2020-06-16T17:18:09Z"
+    "last-modified": "2020-06-22T18:38:34Z"
   },
   "releases": [
     {
-      "version": "32.20200615.1.0",
+      "version": "32.20200615.1.2",
       "metadata": {
         "rollout": {
-          "start_epoch": 1592335800,
+          "start_epoch": 1595451600,
```

Comment:
> next rollout of 32.20200615.1.2 scheduled for 2020-07-22 21:00:00 UTC for 2880m (48.0h)

Hmm, that should be 2020-06-22, right?

Reply:
And looks like the same issue with https://github.com/coreos/fedora-coreos-streams/pull/134

Reply:
yeah, let me correct that detail.